meta-lmp-base/recipes-samples/images/lmp-image-common.inc

```diff
@@ -13,8 +13,13 @@ SRC_URI = "\
     file://sudoers \
 "
 
-# Enough free space for a full image update
-IMAGE_OVERHEAD_FACTOR = "2.3"
+# In most cases the rootfs will be expanded to fit available storage
+# However, for QEMU targets we need to pre-allocate enough free space
+# for a full image update
+IMAGE_OVERHEAD_FACTOR_qemuriscv64 ?= "2.3"
+IMAGE_OVERHEAD_FACTOR_qemuarm64 ?= "2.3"
+IMAGE_OVERHEAD_FACTOR_qemuarm ?= "2.3"
+IMAGE_OVERHEAD_FACTOR_intel-corei7-64 ?= "2.3"
```

Comment:
I understand the QEMU changes here, but what is the rationale behind this target? It should expand properly like our other hardware based MACHINES?

Reply:
We spoke on Slack but for the purposes of documenting:
This is here because we can use the intel-corei7-64 WIC image with QEMU x86 and in that use-case you still need the larger space (similar to the other QEMU targets).